app/javascript/src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 
+import "src/utils/dayjs"; //eslint-disable-line
```

Comment:
Go through https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F very well and make necessary changes.

The point of writing and documenting standards in the React Best Practices book is to ensure that we avoid redundant review comments like the one I am giving right now.

If you don't agree with what's mentioned in the book, then it's totally fine and it's acceptable to keep this change. But you ought to initiate a discussion and make your strong points against the content from the book. If there's a need to update the content in the book then we can do that.

If you feel the content in the book makes sense and if you are not following those standards in your code, then that's not acceptable. I understand that folks miss things out. Thus the self-review process. Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Reply:
What led me to keep this in utils instead of lib was the thought that this would clean the overall directory structure since there wasn't anything else in the lib sub-directory. I will look into that further.

Reply:
I understood the issue and changed it. I also renamed the date format functions as wasCreatedAtFormat and wasCreatedAgoFormat respectively to make the name more obvious. I hope this naming convention was okay.

Reply:
No need to couple `createdAt` with the util functions. Keep the util functions generic.

Something like `formatWithDayAndDate`, `formatWithShortDate` or `formatWithLongDate` etc are examples of generic names.